tensorflow/lite/micro/tools/make/download_and_extract.sh

```diff
@@ -86,17 +86,6 @@ build_embarc_mli() {
   gmake -j 4 -C ${1}/lib/make TCF_FILE=${2}
 }
 
-setup_zephyr() {
-  command -v virtualenv >/dev/null 2>&1 || {
-    echo >&2 "The required 'virtualenv' tool isn't installed. Try 'pip install virtualenv'."; exit 1;
-  }
-  virtualenv -p python3 ${1}/venv-zephyr
-  . ${1}/venv-zephyr/bin/activate
-  python ${1}/venv-zephyr/bin/pip install -r ${1}/scripts/requirements.txt
-  west init -m https://github.com/zephyrproject-rtos/zephyr.git
-  deactivate
-}
-
```

Comment:
Should we delete this? Not adapt to clone our repo or something like this?

Reply:
I have deleted this because I assumed every one of us will do the first steps from zephyr getting started guide. If so there is no need to keep these lines and I've done that to keep the codebase nice&clean.